core/build.gradle

```diff
@@ -290,7 +290,7 @@ task('verifyVersions') {
     }
     Set<String> knownVersions = new TreeSet<>(xml.versioning.versions.version.collect { it.text() }.findAll { it ==~ /\d\.\d\.\d/ })
```

Comment:
It is weird to me this task is on core. Could it just be in the root project? And create a precommit task there which depends on this?